src/fides/core/config/helpers.py

```diff
@@ -103,41 +101,6 @@ def update_config_file(  # type: ignore
             echo(f"\tSet {key}.{subkey} = {val}")
 
 
-def create_config_file(
```

Comment:
this was moved to `docs.py` since the docs version is the only version that should be exported